include/ros1_bridge/factory.hpp

```diff
@@ -152,6 +152,92 @@ class Factory : public FactoryInterface
     ROS1_T & ros1_msg);
 };
 
+template <class R1_TYPE, class R1_REQ, class R1_RES, class R2_TYPE, class R2_REQ, class R2_RES>
+class ServiceFactory : public ServiceFactoryInterface
+{
+private:
+
+  void translate_1_to_2(const R1_REQ&, R2_REQ&);
+  void translate_1_to_2(const R1_RES&, R2_RES&);
+  void translate_2_to_1(R1_REQ&, const R2_REQ&);
+  void translate_2_to_1(R1_RES&, const R2_RES&);
```

Comment:
Please swap the argument order to match the name of the functions. Might become obsolete with the comment above.